pkg/driver/controller.go

```diff
@@ -166,10 +166,20 @@ func (d *Driver) ValidateVolumeCapabilities(ctx context.Context, req *csi.Valida
 		return nil, status.Errorf(codes.Internal, "Could not get volume with ID %q: %v", volumeID, err)
 	}
 
-	supported := d.isValidVolumeCapabilities(volCaps)
-	return &csi.ValidateVolumeCapabilitiesResponse{
-		Supported: supported,
-	}, nil
+	confirmed := d.isValidVolumeCapabilities(volCaps)
+	if confirmed {
+		return &csi.ValidateVolumeCapabilitiesResponse{
+			Confirmed: &csi.ValidateVolumeCapabilitiesResponse_Confirmed{
+				// TODO if volume context is provided, should validate it too
+				// VolumeContext:      req.GetVolumeContext(),
+				VolumeCapabilities: volCaps,
+				// TODO if parameters are provided, should validate them too
```

Comment:
I have left 2 TODOs here for a follow-up PR. CreateVolume could use the same validation